docs/05-plugin-development.md

```diff
@@ -225,20 +225,21 @@ You can use [`this.getModuleInfo`](guide/en/#thisgetmoduleinfomoduleid-string--m
 #### `watchChange`
 Type: `(id: string) => void`<br>
 Kind: `sync, sequential`<br>
-Previous/Next Hook: This hook can be triggered at any time both during the build and the output generation phases.
+Previous/Next Hook: This hook can be triggered at any time both during the build and the output generation phases. If that is the case, the current build will still proceed but a new build will be scheduled to start once the current build has completed, starting again with [`options`](guide/en/#options).
```

Comment:
This is great! I was never sure if build re-runs were sometimes shortcut. This makes it clear.